### ISSUE TITLE
Unable to delete workspace (if deleted in Azure Portal)

### ISSUE THREAD

**Issue by trbran** [Microsoft/vscode-tools-for-ai#186]
Issue Type: <b>Bug</b>

If users nginx/1.15.2 token is larger than 10240 bytes, both View Pipeline and View Experiment will fail with a '400 Bad Request, Request header or cookie too large'.

Extension version: 0.4.2
VS Code version: Code - Insiders 1.32.0-insider (43228c493069ead7cfb7d57b2da1db20fc0acdaf, 2019-03-06T01:28:07.880Z)
OS version: Windows_NT x64 10.0.17763

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz (8 x 2112)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|15.84GB (7.06GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|

</details>

**Comment by greazer** [comment]
This may be because you have too many active subscriptions. Please let me know if by unselecting several subscriptions this problem goes away.

**Comment by trbran** [comment]
Actually, it’s not the number of subscriptions, but rather the number of groups.

We just found the same to be the case in AzureML Pipelines (Trond Brande: Issue related to ...ensure_default_datatypes() in SDK 1.0.17? <https://teams.microsoft.com/l/message/19:7836c9f6e6e4439baae5ea6bda1f3a68@thread.skype/1551722771525?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=88aa174e-6310-4634-bfcb-5761e1a1190a&parentMessageId=1551722771525&teamName=ML%20Platform&channelName=Pipeline%20(aeva)&createdTime=1551722771525> ) where it was fixed ‘some people have very large auth headers due to many or large claims, our ae3pservice nginx container has a request header max size limit. Adjusting the nginx configurations to allow larger headers fixes the issue’

Decoding the token shows that subscriptions takes very little space, but the fact that I have 139 groups is what takes place,

  "groups": [
    "000c5803-9a71-486b-8b05-6eee8b4e5835",
    "ba908941-18fe-4655-ac5d-ec2fa5fb8f46",
    "ee5da26a-d856-47e1-bc79-8f05e0dec8e7",

Regards,
Trond
